src/Mod/Fem/Gui/ViewProviderFemConstraint.cpp

```diff
@@ -115,6 +123,32 @@ void ViewProviderFemConstraint::attach(App::DocumentObject* pcObject)
     addDisplayMaskMode(sep, "Base");
 }
 
+std::string ViewProviderFemConstraint::resourceSymbolDir =
+    App::Application::getResourceDir() + "Mod/Fem/Resources/symbols/";
+
+void ViewProviderFemConstraint::loadSymbol(const char* fileName)
+{
+    ivFile = fileName;
+    SoInput* in = new SoInput;
+    in->openFile(ivFile);
+    SoSeparator* nodes = SoDB::readAll(in);
```

Comment:
If for some reason something goes wrong with the loading then `nodes` will be a null pointer. This should be checked before de-referencing it.

Reply:
Done